lib/glific/partners.ex

```diff
@@ -929,6 +929,11 @@ defmodule Glific.Partners do
     {:ok, credential}
   end
 
+  defp credential_update_callback(organization, credential, "maytapi") do
+    WhatsappGroup.list_wa_groups(organization.id)
```

Comment:
Here we fetch wa groups and add to groups table on credential update callback. But there can be unique constraint error (`[:label, :organization_id]`). Should we do an upsert in this case or we just error out in this case?

Reply:
> Here we fetch wa groups and add to groups table on credential update callback. But there can be unique constraint error (`[:label, :organization_id]`). Should we do an upsert in this case or we just error out in this case?

This works fine, redundant entries returns changeset error, since we are not doing bulk insert, there is no db revert and new entries will be added fine